src/opts.h

```diff
@@ -45,7 +46,8 @@ enum {
 	KEY_NOINITGROUPS,
 	KEY_RELAXED_PERMISSIONS,
 	KEY_STATFS_OMIT_RO,
-	KEY_VERSION
+	KEY_VERSION,
```

Comment:
please leave KEY_VERSION at the end. move KEY_DIRECTO_IO to between KEY_CHROOT and KEY_COW (just as you did in your "switch" code).